Dockerfile

```diff
@@ -1,9 +1,9 @@
-# Use singlecellportal/phusion_passenger-full as base image. To make your builds reproducible, make
+# Use singlecellportal/phusion_passenger-ruby32 as base image. To make your builds reproducible, make
 # sure you lock down to a specific version, not to `latest`!
 # See https://github.com/phusion/passenger-docker/blob/master/Changelog.md for
 # a list of version numbers.
 
-FROM singlecellportal/phusion_passenger-full:2.4.1
+FROM singlecellportal/phusion_passenger-ruby32:2.5.1
```

Comment:
The section on "Production-ready YJIT" in the [Ruby 3.2 release notes](https://www.ruby-lang.org/en/news/2022/12/25/ruby-3-2-0-released/) caught my eye, particularly:

> Overall YJIT is 41% faster (geometric mean) than the Ruby interpreter on yjit-bench.

In my cursory glance it looks like [YJIT](https://github.com/ruby/ruby/blob/master/doc/yjit/yjit.md) isn't on by default, but that'd be nice some day!  I also don't have a good sense of whether YJIT would noticeably speed up page reloads when frontend Vite SW caching applies less (e.g. upload frontend development), which is the only area that comes to mind for where faster Ruby would substantially help.

Reply:
Interesting!  That could be good to check out.  However, given that we're moving to pre-rendering everything, I think this is potentially moot as we wouldn't be caching most things server-side anymore.